data/abilities.ts

```diff
@@ -2666,7 +2666,7 @@ export const BattleAbilities: {[abilityid: string]: AbilityData} = {
 		desc: "This Pokemon's damaging moves become multi-hit moves that hit twice. The second hit has its damage quartered. Does not affect multi-hit moves or moves that have multiple targets.",
 		shortDesc: "This Pokemon's damaging moves hit twice. The second hit has its damage quartered.",
 		onPrepareHit(source, target, move) {
-			if (move.category === 'Status' || move.selfdestruct || move.multihit) return;
+			if (move.category === 'Status' || move.selfDamage?.startsWith("KO") || move.multihit) return;
```

Comment:
Yeah, I've seen `move.selfDamage?.startsWith("KO")` in enough places to think we shouldn't have merged `selfdestruct` and `selfDamage`. Can we split them back out?